### PR TITLE
Hungarian translation

### PR DESCRIPTION
**Pull request by cinadr**
Hi!

I've just finished hungarian translation for Cairo.
Hope you're interested.

Happy New Year!
Regards,
Zsolt.

### CODE REVIEW

File: Cairo Desktop/CairoDesktop.Localization/DisplayString.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ public class DisplayString
             new KeyValuePair<string, string>("Svenska", "sv_SE"),
             new KeyValuePair<string, string>("Turkish (Türkçe)", "tr_TR"),
             new KeyValuePair<string, string>("한국어", "ko_KR")
```
Comment (dremin) [inline Cairo Desktop/CairoDesktop.Localization/DisplayString.cs:22]: Could you please add a comma to the end of this line to fix the build error?

**Review by dremin — CHANGES_REQUESTED** [review]
Thanks, happy new year! If you could please fix the 1 issue causing an error, I'll then merge!

File: Cairo Desktop/CairoDesktop.Localization/DisplayString.cs (hunk 1 above)
Comment (cinadr) [reply Cairo Desktop/CairoDesktop.Localization/DisplayString.cs]: Done.

**Review by dremin — APPROVED** [review]
Thanks!

**Pull request merged**